src/Microsoft.AspNetCore.Mvc.Razor.Host/MvcRazorCompilation.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.Razor.Internal;
+using Microsoft.AspNetCore.Razor.Evolution;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Compilation
+{
+    public class MvcRazorCompilation : RazorCompilation
```

Comment:
The code that's in this class seems right, but it shouldn't be coupled to a specific item.